CHANGELOG.md

```diff
@@ -1,3 +1,23 @@
+x.x.x Release notes (yyyy-MM-dd)
+=============================================================
+NOTE: This version bumps the Realm file format to version 11. It is not possible to downgrade version 10 or earlier. Moreover, older files will automatically be upgraded to the new file format. Files created by Realm JavaScript prior to v1.0.0, might not be upgradeable. 
+
+### Enhancements
+* None.
+
+### Fixed
+* Upgrading files with string primary keys would result in a file where it was not possible to find the objects by primary key. ([realm/realm-cocoa#6716](https://github.com/realm/realm-cocoa/issues/6716), since v6.0.4)
```

Comment:
This is a core version number and not a JS version number.

Reply:
The core version is v6.0.7 (https://github.com/realm/realm-core/pull/3893/) while v6.0.4 refers to the corresponding JS version. Let me it a bit clearer.